### PR TITLE
Added support for EXIT op

### PR DESCRIPTION
**Pull request by radrow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Support for `EXIT` opcode via `exit : (string) => 'a` function (behaves same as `ABORT`, but consumes all gas).
```
Comment (ghallak) [inline CHANGELOG.md:9]: I usually append new features to the end of the `Added` list in the change log.

**Comment by ghallak** [comment]
Can you add this to the docs in the section about exceptions?

File: CHANGELOG.md (hunk 1 above)
Comment (radrow) [reply CHANGELOG.md:9]:
```suggestion
- Added support for `EXIT` opcode via `exit : (string) => 'a` function (behaves same as `ABORT`, but consumes all gas).
```

**Review by ghallak — APPROVED** [review]

**Review by hanssv — APPROVED** [review]
Not sure this is super productive, but LGTM.

**Comment by marc0olo** [comment]
@radrow any examples where this is useful? just wondering :)

**Comment by radrow** [comment]
Not really, but since we have this in fate and there was epsilon cost to add it...

**Pull request merged**